cg_hermes/config/raredisease.py

```diff
@@ -0,0 +1,147 @@
+"""Tags that are defined in Tomte deliverables mapped to tags used in CG
+
+The tag sets that exists in all files are set to mandatory. Tag sets that exists in < 4 deliverables are not
+mandatory by default. However, the tags that are available to a particular analysis is mandatory for that analysis.
+"""
+
+from cg_hermes.config.nextflow import NEXTFLOW_TAGS
+
+
+RAREDISEASE_COMMON_TAGS = {
+    frozenset(["chromograph_upd", "sites"]): {
+        "tags": ["chromograph", "upd", "sites"],
+        "is_mandatory": False,
+        "used_by": ["scout"],
+    },
+    frozenset(["chromograph_upd", "regions"]): {
+        "tags": ["chromograph", "upd", "regions"],
+        "is_mandatory": False,
+        "used_by": ["scout"],
+    },
+    frozenset(["chromograph_cov", "tcov"]): {
+        "tags": ["chromograph", "tcov"],
+        "is_mandatory": False,
+        "used_by": ["scout"],
+    },
+    frozenset(["chromograph_rhoviz", "autozyg"]): {
+        "tags": ["chromograph", "autozyg"],
+        "is_mandatory": False,
+        "used_by": ["scout"],
+    },
+    frozenset(["rank_and_filter", "clinical"]): {
+        "tags": ["vcf-snv-clinical"],
+        "index_tags": ["vcf-snv-clinical-index"],
```

Comment:
there was a discussion about standarising HK tags for workflows. @ivadym @ChrOertlin any opinions for these?

Reply:
We should start using separate tags, i.e: ["vcf", "snv", "clinical", "index"],

Reply:
We would need make a decision here and to document it (not quiet there yet). But I guess we'll have a decision before the pipeline is in production, so it can even be `vcf-snv-clinical-index` for now

Reply:
tomte is planned to go in production next week. So you are aware. But I guess we could update it later

Reply:
Keeping MIP matching for now